aether-ui/aether/ui/assets/apps/components/NavBar.jsx

```diff
@@ -39,7 +39,7 @@ class NavBar extends Component {
 
     return (
       <div data-qa='navbar' className='navbar top-nav'>
-        <Link className='top-nav-logo' title='aether' to='#'>
+        <Link className='top-nav-logo' title='aether' onClick={this.props.cb}>
```

Comment:
`to` is mandatory.... we need to find a way to solve this point :disappointed: 

https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/Link.md